### ISSUE TITLE
available active networks check error

### ISSUE THREAD

**Issue by jennafarr** [warfare-plugins/social-warfare#290]
in functions/share-count-function.php.

It looks like there is already an attempt to remove "active" from the list of networks. This snippet does not have any effect:

if(isset($networks['active'])) { unset($networks['active']); }

Instead, above that, as soon as we grab the available networks we handle the check:

$availableNetworks = $options['newOrderOfIcons'];
$networks = array();

if ($availableNetworks) {
if(isset($availableNetworks['active'])) { unset($availableNetworks['active']); }

foreach ( $availableNetworks as $key => $value ) :
if ( $options['newOrderOfIcons'][ $key ] ) {
$networks[] = $key;
}
endforeach;
}

This modification allows the counts to be performed without an error being logged.

**Comment by Nicholas-Cardot** [comment]
I'm pushing an update for this issue now.

Instead of detecting it in the share-count-function.php, I'm going to attempt to stop it at it's source. There are currently two places where that index seems to popup in that array. 

1. It can come when calling the user options via swp_get_user_options();
Solution: I added a line at the end of that function to filter out that index if it exists prior to returning it's final array.

2. It can also come when calling the available buttons filter via apply_filters( 'swp_button_options' , $icons_array );
Solution: I added a filter with priority 500 (so it runs after all other filters) that checks for the index and strips it out prior to that function returning it's final array.

This should solve that notice fully in all instances and prevent it from reappearing anywhere else.

Does that make sense?
